src/plays/calendar/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Calendar
```

Comment:
You can add a few more implementation details here. This read me will be used by people to understand the technicalities of the play a bit better. Consider it like a blog for someone to read to know how this play was built. 😀